src/base/task.cc

```diff
@@ -851,7 +855,7 @@ void TaskScheduler::WaitForTerminateCompletion() {
 
     int count = 0;
     int threadsRunning;
-    pid_t pid = getpid();
+    pid_t pid = osspecific_getpid();
```

Comment:
This breaks the compatibility with Linux and other versions. Please either implement` getpid()`  for Windows(recommended) or add `osspecific_getpid` for Linux, etc.